src/components/layouts/Page/index.js

```diff
@@ -33,10 +40,12 @@ const PageContent = ({
 
 const Page = ({
   children,
+  contentContainerStyles,
   hasBackButton = true,
   isHome,
   onBackButtonClick,
   currentProgress,
+  rootContainerStyles,
```

Comment:
Adding style overrides for the Page container and the content container within the page.